src/panels/lovelace/cards/hui-iframe-card.ts

```diff
@@ -0,0 +1,86 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/ha-card.js";
+
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+interface Config extends LovelaceConfig {
+  aspect_ratio?: string;
+  title?: string;
+  url: string;
+}
+
+export class HuiIframeCard extends HassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  protected config?: Config;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    return 1 + this.offsetHeight / 50;
+  }
+
+  public setConfig(config: Config) {
+    if (!config.url) {
+      throw new Error("URL required");
+    }
+
+    this.config = config;
+    this.requestUpdate();
```

Comment:
Realize we don't need to do this. `config` is already a property so this is happening automatically when we do `this.config = config`. I realize that we need to update this in other cards too.